.cirrus.yml

```diff
@@ -186,35 +180,21 @@ task:
         - fi
         - export CIRRUS_CHANGE_MESSAGE=`cat /tmp/cirrus_change_message.txt`
         - export CIRRUS_COMMIT_MESSAGE=`cat /tmp/cirrus_commit_message.txt`
-    ### Linux desktop tasks ###
-    - name: build-linux+drive-examples
-      env:
-        matrix:
-          CHANNEL: "master"
-          CHANNEL: "stable"
-      build_script:
-        - flutter channel $CHANNEL
-        - ./script/incremental_build.sh build-examples --linux
-        - xvfb-run ./script/incremental_build.sh drive-examples --linux
 
 task:
   # Xcode 12 task
```

Comment:
Would you mind removing this comment while you're here?  I should have removed it with #3653.

Reply:
Done.